settings.gradle

```diff
@@ -2,6 +2,10 @@ rootProject.name = 'usi'
 
 include 'interfaces'
 include 'core'
+include 'metrics'
+findProject(':metrics')?.name = 'usi-metrics'
+include 'metrics-dropwizard'
+findProject(':metrics-dropwizard')?.name = 'usi-metrics-dropwizard'
```

Comment:
What's the output of `./gradlew projects`? What do you think about setting the project name in `subprojects`?

Reply:
```
$ gradle projects
Root project 'usi'
+--- Project ':core'
+--- Project ':hello-world'
+--- Project ':interfaces'
+--- Project ':usi-metrics'
\--- Project ':usi-metrics-dropwizard'
```

Reply:
I removed it completely